com.github.danrabbit.harvey.yml

```diff
@@ -0,0 +1,24 @@
+id: com.github.danrabbit.harvey
+runtime: org.gnome.Platform
+runtime-version: '3.28'
+sdk: org.gnome.Sdk
+command: com.github.danrabbit.harvey
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--talk-name=ca.desrt.dconf'
+  - '--env=DCONF_USER_CONFIG_DIR=.config/dconf'
+  - '--filesystem=xdg-run/dconf'
+  - '--filesystem=~/.config/dconf:ro'
+cleanup:
+  - /include
+modules:
+  - name: harvey
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://github.com/danrabbit/harvey/archive/1.0.0.tar.gz
+        sha256: 51d8e5d8e6eab405a713dcb635bfbff75c3187fdf8a0c5e16d5bec400a81a758
+      - type: patch
+        path: elementary-theme.patch
```

Comment:
Regarding this – you probably want to import [this](https://github.com/flathub/com.github.robertsanseries.ciano/blob/master/com.github.robertsanseries.ciano.json#L83), since if someone runs it without having the theme installing, it’s going to look like this:

![obrazok](https://user-images.githubusercontent.com/7472425/45925745-b797c800-bf1b-11e8-8378-27e972ae0256.png)

Reply:
> Regarding this – you probably want to import [this](https://github.com/flathub/com.github.robertsanseries.ciano/blob/master/com.github.robertsanseries.ciano.json#L83), since if someone runs it without having the theme installed, it’s going to look like this:

im not sure on what issue you are referred here, the broken GTK theme or the broken icons? tbh i dont think we can do much here, unless @danrabbit wants to support GNOME styles, and in general this is an issue with more elementary apps on Flathub

i guess elementary apps on Flathub is mostly for the people who run Patheon desktop on Fedora or Arch or openSUSE etc, and they dont have access to elementary AppCenter repository

Reply:
btw this is how it looks like with elementary Flatpak GTK and elementary icons (system) installed 

![screenshot from 2018-09-23 11 58 32](https://user-images.githubusercontent.com/11244308/45926155-2bd86880-bf28-11e8-92ce-160651ce6caf.png)

Reply:
I’m talking about that to make it look good on other systems, you can bundle in the theme. The app will use it with the patch you added.

The icons I’m not sure.